workspaces/liferay-sample-workspace/client-extensions/liferay-sample-custom-checkout-step/client-extension.yaml

```diff
@@ -1,17 +1,26 @@
 assemble:
     - fromTask: bootJar
+    - from: src/resources/assets
```

Comment:
<img width="1911" alt="Screenshot 2023-12-12 at 18 16 42" src="https://github.com/liferay-commerce/liferay-portal/assets/580733/2f1c2030-db72-4219-a08a-7fb29ba04317">

Reply:
This is failing because you cannot use bare specifiers in imports unless you define an import map, which is **probably** something you don't want to do here unless you have good reasons for it.